e2e/playwright/search/src/tests/search-sorting.spec.ts

```diff
@@ -0,0 +1,148 @@
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { ApiClientFactory, Utils, test, TrashcanApi, NodesApi, FileActionsApi, TEST_FILES, SearchPage } from '@alfresco/playwright-shared';
+
+test.describe.only('Search sorting', () => {
+  const random = Utils.random();
+
+  const user1 = `user1-${random}`;
+  const user2 = `user2-${random}`;
+
+  const parent = `parent-${random}`;
+  let parentId: string;
+
+  const fileJpg = {
+    name: `search-sort-${random}-file-1.jpg`,
+    source: TEST_FILES.JPG_FILE.path
+  };
+
+  const filePdf = {
+    name: `search-sort-${random}-file-2.pdf`,
+    title: 'search sort title',
+    description: 'search-sort-${random}-file',
+    source: TEST_FILES.PDF.path
+  };
+
+  let nodesApi1: NodesApi;
+  let trashcanApi1: TrashcanApi;
+  let fileActionsApi1: FileActionsApi;
+  let nodesApi2: NodesApi;
+  let trashcanApi2: TrashcanApi;
+  let fileActionsApi2: FileActionsApi;
+
+  test.beforeAll(async () => {
+    try {
+      const apiClientFactory = new ApiClientFactory();
+      await apiClientFactory.setUpAcaBackend('admin');
+      await apiClientFactory.createUser({ username: user1 });
+      await apiClientFactory.createUser({ username: user2 });
+      nodesApi1 = await NodesApi.initialize(user1, user1);
+      trashcanApi1 = await TrashcanApi.initialize(user1, user1);
+      fileActionsApi1 = await FileActionsApi.initialize(user1, user1);
+      nodesApi2 = await NodesApi.initialize(user2, user2);
+      trashcanApi2 = await TrashcanApi.initialize(user2, user2);
+      fileActionsApi2 = await FileActionsApi.initialize(user2, user2);
+    } catch (error) {
+      console.error(`beforeAll failed : ${error}`);
+    }
+
+    parentId = (await nodesApi1.createFolder(parent)).entry.id;
+
+    await nodesApi1.setGranularPermission(parentId, true, user2, 'Collaborator');
+
+    await fileActionsApi1.uploadFileWithRename(fileJpg.source, fileJpg.name, parentId);
+    await fileActionsApi2.uploadFileWithRename(filePdf.source, filePdf.name, parentId, filePdf.title, filePdf.description);
+
+    await fileActionsApi1.waitForNodes(fileJpg.name, { expect: 1 });
+    await fileActionsApi2.waitForNodes(filePdf.name, { expect: 1 });
+  });
+
+  test.beforeEach(async ({ loginPage }) => {
+    await Utils.tryLoginUser(loginPage, user1, user1, 'beforeEach failed');
+  });
+
+  test.afterAll(async () => {
+    await Utils.deleteNodesSitesEmptyTrashcan(nodesApi1, trashcanApi1, 'afterAll failed');
+    await Utils.deleteNodesSitesEmptyTrashcan(nodesApi2, trashcanApi2, 'afterAll failed');
+  });
+
+  async function testSearchSorting(
+    searchPage: SearchPage,
+    sortBy: 'Relevance' | 'Title' | 'Filename' | 'Modified date' | 'Modifier' | 'Created date' | 'Size' | 'Type',
```

Comment:
you have the same type for sortBy in search-sorting-picker.component.ts - instead of duplicating these all values could you please make it reusable? You can export that type and use here as well.